client/src/components/Button/Button.css

```diff
@@ -1,3 +1,81 @@
-button {
-	background-color: red;
+.button-light-pink {
+    border: 1px;
+    background-color: #FAE5DC;
+    color: #441F0E;
+    font-size: 15px;
+    font-weight: bold;
+    line-height: 22px;
+    font-style: normal;
+    font-family: Sahitya;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    height: 31px;
+    width: 88px;
+   
 }
```

Comment:
Can you simplify this CSS? There's a lot of shared styles that could be better organized